gcp/api/server.py

```diff
@@ -1169,13 +1196,12 @@ def _query_by_comparing_versions(context: QueryContext, query: ndb.Query,
         context.total_responses.add(1)
         break
 
-  return bugs, cursor
+  return bugs
 
 
 @ndb.tasklet
 def query_by_package(context: QueryContext, package_name: str, ecosystem: str,
-                     purl: PackageURL | None,
-                     to_response: Callable) -> tuple[list, ndb.Cursor]:
+                     purl: PackageURL | None, to_response: Callable) -> list:
   """Query by package."""
```

Comment:
Please improve this docstring while you're here: https://google.github.io/styleguide/pyguide.html#s3.8.3-functions-and-methods